### PR TITLE
Added/Modified Scripts to verify OpenJDK builds

### PR DESCRIPTION
**Pull request by Willsparker**
Added into the script that builds the vagrant machines to test the playbooks, the scripts to run and build openJDk on the machines, to ensure the ansible playbook correctly installed everything.

### CODE REVIEW

**Comment by Willsparker** [comment]
Relates to #842

File: ansible/pbTestScripts/buildJDK.sh [hunk 1]
```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```
Comment (karianna) [inline ansible/pbTestScripts/buildJDK.sh]: This seems very hardcoded to one particular variation.  If that's on purpose then that's OK but perhaps rename the file to reflect reality.  Else you can parameterize this script and pass arguments into it...

File: ansible/pbTestScripts/testScript.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
 set -eu
 
 # Takes all arguments from the script
-processArgs()
+processArgs()
```
Comment (karianna) [inline ansible/pbTestScripts/testScript.sh]: whitespace

File: ansible/pbTestScripts/testScript.sh [hunk 3]
```diff
@@ -31,19 +31,22 @@ setupGit()
...
 	cd ~/adoptopenjdkPBTests/$2/ansible
-
-	# Alias the correct vagrant file
-	ln -sf Vagrantfile.$1 Vagrantfile
-	
-	vagrant up
-
-	# Remotely moves to the correct directory in the VM and builds the playbook. Then logs the VM's output to a file, in a separate directory
+	ln -sf Vagrantfile.$1 Vagrantfile	#Alias the correct vagrant file
+	vagrant  up
```
Comment (karianna) [inline ansible/pbTestScripts/testScript.sh]: whitespace

File: ansible/pbTestScripts/testScript.sh [hunk 4]
```diff
@@ -31,19 +31,22 @@ setupGit()
...
 	vagrant ssh -c "cd /vagrant/playbooks/AdoptOpenJDK_Unix_Playbook && sudo ansible-playbook --skip-tags "adoptopenjdk,jenkins" main.yml" 2>&1 | tee ~/adoptopenjdkPBTests/logFiles/$2.$1.log
 
+		# ^ Remotely moves to the correct directory in the VM and builds the playbook. Then logs the VM's output to a file, in a separate directory
```
Comment (karianna) [inline ansible/pbTestScripts/testScript.sh]: Prefer comments above the line of code that it's executing, also - whitespace.

File: ansible/pbTestScripts/testScript.sh [hunk 5]
```diff
@@ -75,16 +78,16 @@ setupFiles
 setupGit $1 $folderName
 
 # For all tested OSs / Playbooks
-for OS in Ubuntu1804 Ubuntu1604 CentOS6
+for OS in Ubuntu1804 Ubuntu1604 CentOS6
```
Comment (karianna) [inline ansible/pbTestScripts/testScript.sh]: whitespace.

File: ansible/pbTestScripts/buildJDK.sh (hunk 1 above)
Comment (sxa555) [reply ansible/pbTestScripts/buildJDK.sh]: Agree, although I've got so many other enhancements I would like to get on that I'd quite like to leave this as-is for now then adjust later if you're OK with that Martijn. I may even go for a random selection later. This is basically a simp!eat form of what's needed to run our builds using our environment scripts from openjdk-build. We're hoping to do something similar for running a basic operation fr. The test suites too

Comment (karianna) [reply ansible/pbTestScripts/buildJDK.sh]: OK :-)

**Pull request merged**